test/sparsedir/cholmod.jl

```diff
@@ -185,10 +190,6 @@ ACSC = sprandn(10, 10, 0.3) + I
 @test ishermitian(Sparse(Hermitian(complex(ACSC), :L)))
 @test ishermitian(Sparse(Hermitian(complex(ACSC), :U)))
 
-# test Sparse constructor for c_Sparse{Tv,Ti} input( and Sparse*Sparse)
-B = CHOLMOD.Sparse(SparseMatrixCSC{Float64,Int32}(sprandn(48, 48, 0.1))) # A has Int32 indices
```

Comment:
Would it be better to keep this test but change `Int32` to `SuiteSparse_long` ? Do we have `sparse(A) * sparse(B)` covered anywhere else?